lib/TPP/GPU/GpuVectorize.cpp

```diff
@@ -29,6 +29,7 @@
 #include "mlir/Transforms/Passes.h"
 
 using namespace mlir;
+using namespace mlir::vector;
```

Comment:
Why is this here? Please keep the unrelated changes for further PRs. This carries the `mlir::vector` changes everywhere that are not necessary.

Reply:
Its not an unrelated change, it is a build fix, vector is not disambiguated otherwise. std::vector and mlir:vector are both visible to the .cpp file. We need to explicitly make mlir::vector visible for build to go through.

Reply:
This should only happen if both `std` and `mlir` namespaces are in scope. You should not have `using namespace std` in LLVM source files.